frontend/vue/components/CodeExercise/CodeOutput.vue

```diff
@@ -0,0 +1,88 @@
+<template>
+  <div ref="outputDiv" class="code-output">
+    <div class="code-output__state-info__wrapper">
```

Comment:
Unused `code-output__state-info__wrapper` class:

```suggestion
    <div>
```